File: src/main/java/com/itspub/core/ResultSetHandler.java
package com.itspub.core;

import java.sql.ResultSet;
import java.sql.SQLException;
import java.util.List;

/**
 * Created by Administrator on 2016/12/15.
 */
public interface ResultSetHandler<T> {
    List<T> doWitResultSet(ResultSet rs) throws SQLException;
}

File: src/main/kotlin/com/itspub/util/ResultSetHandler.kt
package com.itspub.util

import java.sql.ResultSet
import java.sql.SQLException

/**
 * Created by Administrator on 2016/12/29.
 */
interface ResultSetHandler<T> {
    @Throws(SQLException::class)
    fun doWithResultSet(rs: ResultSet): List<T>
}